name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/BitfinexQuoteFeed.java

```diff
@@ -0,0 +1,201 @@
+package name.abuchen.portfolio.online.impl;
+
+import java.io.IOException;
+import java.text.MessageFormat;
+import java.text.ParseException;
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONValue;
+
+import name.abuchen.portfolio.Messages;
+import name.abuchen.portfolio.model.Exchange;
+import name.abuchen.portfolio.model.LatestSecurityPrice;
+import name.abuchen.portfolio.model.Security;
+import name.abuchen.portfolio.model.SecurityPrice;
+import name.abuchen.portfolio.online.QuoteFeed;
+import name.abuchen.portfolio.util.WebAccess;
+
+public final class BitfinexQuoteFeed implements QuoteFeed
+{
+
+    public static final String ID = "BITFINEX"; //$NON-NLS-1$
+
+    @Override
+    public String getId()
+    {
+        return ID;
+    }
+
+    @Override
+    public String getName()
+    {
+        return "Bitfinex Cryptocurrency Exchange"; //$NON-NLS-1$
```

Comment:
Please consider to localize the name like it is done in https://github.com/buchen/portfolio/blob/004996d700cca625ed8674ca90bbe2708ffc021e/name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/YahooFinanceQuoteFeed.java#L56